src/components/login/loginForm/LoginForm.jsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Formik } from 'formik';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import Button from '@material-ui/core/Button';
+import './loginForm.scss';
+import * as Yup from 'yup';
+import { useDispatch, useSelector } from 'react-redux';
+import preloader from '../../../assets/gif/preloader.gif';
+import { fetchLoginData } from '../../../store/actions/loginActions';
+
+// filtering and checking what the user has entered into forms
+const validationSchema = Yup.object().shape({
+	email: Yup.string()
+		.email('Invalid email')
+		.required('Required'),
+	password: Yup.string()
+		.required('Required')
+});
+
+const LoginForm = () => {
+
+	const loading = useSelector((state) => state.login.loading);
+
+	const dispatch = useDispatch();
+
+	// styles from material ui
+	const useStyles = makeStyles((theme) => ({
+		root: {
+			'& > *': {
+				margin: theme.spacing(1),
+				width: '25ch',
+			},
+		},
+	}));
+	const classes = useStyles();
+
+	// function 'submit' where will be all actions after submitting
+	const submit = (values, { setSubmitting }) => {
+		alert(values.email, values.password);
```

Comment:
remove it. That shouldn't be in develop

Reply:
alert removed
https://github.com/mtalstykh-exadel/team4_frontend/blob/27f79d12b2ded8e5b31bd2b44a4a2d0ffd8adfa8/src/components/login/loginForm/LoginForm.jsx#L38-L42